schema/acq-operation.json

```diff
@@ -1,10 +1,101 @@
 {
-  "id": "acq-.json",
+  "id": "acq-operation.json",
   "$schema": "https://json-schema.org/draft-04/schema#",
   "description": "",
   "type": "object",
-  "example": "../examples/acq-.json",
+  "example": "../examples/acq-operation.json",
   "additionalProperties": false,
-  "required": [],
-  "properties": {}
-}
+  "required": [
+    "operationId",
+    "identifier",
+    "status",
+    "parentOperation"
+  ],
+  "properties": {
+    "operationId": {
+      "type": "string",
+      "description": "a unique identifier for the operation"
+    },
+    "description": {
+      "type": "string",
+      "description": "description of the mission on which the platform observations are made and the objectives of that mission"
+    },
+    "citation": {
+      "$ref": "./citation.json",
+      "description": "identification of the mission"
+    },
+    "identifier": {
+      "$ref": "./identifier.json",
+      "description": "unique identification of the operation"
+    },
+    "status": {
+      "type": "string",
+      "description": "status of the data acquisition",
+      "enum": [
+        "accepted",
+        "completed",
+        "deprecated",
+        "final",
+        "historicalArchive",
+        "notAccepted",
+        "obsolete",
+        "onGoing",
+        "pending",
+        "planned",
+        "proposed",
+        "required",
+        "retired",
+        "superseded",
+        "tentative",
+        "underDevelopment",
+        "valid",
+        "withdrawn"
+      ]
+    },
+    "operationType": {
+      "type": "string",
+      "description": "collection technique for the operation",
+      "enum": [
+        "real",
+        "simulated",
+        "synthesized"
+      ]
+    },
+    "objective": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "object(s) or area(s) of interest to be sensed"
+    },
+    "parentOperation": {
+      "type": "string",
+      "description": "heritage of the operation"
+    },
+    "childOperation": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "sub-missions that make up part of a larger mission"
+    },
+    "plan": {
+      "type": "string",
+      "description": "plan satisfied by the operation"
```

Comment:
@dwalt is it correct that this should be a string? According to [MI_AcquisitionInformation](https://schemas.isotc211.org/19115/-2/gmi/1.0/gmi/#element_MI_AcquisitionInformation) it seems that `acquisitionPlan` should be a class of type [MI_Plan](https://schemas.isotc211.org/19115/-2/gmi/1.0/gmi/#element_MI_Plan)

Reply:
@jwaspin The Operation class references a single Plan instance using an identifier: https://github.com/adiwg/mdJson-schemas/issues/217